src/uikit.cs

```diff
@@ -18159,24 +18255,21 @@ partial interface UIUserNotificationAction : NSCopying, NSMutableCopying, NSSecu
 		[Field ("UIUserNotificationTextInputActionButtonTitleKey")]
 		NSString TextInputActionButtonTitleKey { get; }
 
+#if !WATCH
```

Comment:
Is this intentional?

Reply:
yes, if you look at it, I moved the selector from a partial interface to here. The issue is that I wanted to keep the original version yet keep the interface declaration in a single place.